types/src/proto/events.proto

```diff
@@ -16,7 +16,8 @@ import "proof.proto";
 message Event {
     bytes key = 1;
     uint64 sequence_number = 2;
-    bytes event_data = 3;
+    bytes type_tag = 3;
```

Comment:
We probably shouldn't change the number here to make this change backward compatible. Did I understand it correctly @msmouse?